plugins/com.google.cloud.tools.eclipse.login.test/src/com/google/cloud/tools/eclipse/login/ui/LoginServiceUiTest.java

```diff
@@ -32,6 +31,8 @@
 import org.junit.Assert;
 import org.junit.Test;
 
+import com.google.cloud.tools.eclipse.login.ui.LoginServiceUi.LocalServerReceiverWrapper;
```

Comment:
We haven't really set any rules on import order. I think we have a mix of old and new Google style for this. In any case, done.